crates/relayer-handlers/src/routes/info.rs

```diff
@@ -10,12 +10,34 @@ use webb::evm::ethers::{
 };
 use webb_relayer_context::RelayerContext;
 
+/// Build info data
+#[derive(Debug, Serialize)]
+pub struct BuildInfo {
+    /// Version of the relayer
+    pub version: String,
+    /// Commit hash of the relayer
+    pub commit: String,
+    /// Build time of the relayer
+    pub build_timestamp: String,
```

Comment:
```suggestion
    pub timestamp: String,
```
It reads better in JSON.